src/js/pebble-js-app.js

```diff
@@ -27,8 +31,24 @@ Pebble.addEventListener("appmessage", function(e) {
 });
 
 function downloadBinaryResource(imageURL, callback, errorCallback) {
+
+
+    /*POSTing the image URL to the server for conversion.
+      This conversion is only for Pebble Time. On a original
+      Pebble the dithering should convert the image to have 
+      to 2 colors*/
+    var req1 = new XMLHttpRequest();
+    req1.open("POST",toServer, true);
+    imageURL = "imageurl=" + imageURL;
+    req1.setRequestHeader("Content-type", "imageURL");
+    req1.setRequestHeader("Content-length", imageURL.length);
```

Comment:
You don't include the length of the string `imageURL` - not that this parameter is required in the first place.